### PR TITLE
doc: Breaking change release note for bintools-wrapper

### PR DESCRIPTION
**Pull request by Ericson2314**
###### Motivation for this change

#29396 needs an announcement. 

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: nixos/doc/manual/release-notes/rl-1803.xml [hunk 1]
```diff
@@ -95,6 +95,15 @@ following incompatible changes:</para>
       <link xlink:href="https://search.nix.gsc.io/?q=stateVersion">here</link>.
     </para>
   </listitem>
+  <listitem>
+    <warning>This belongs in Nixpkgs release notes, were we to have it</warning>
```
Comment (Ericson2314) [inline nixos/doc/manual/release-notes/rl-1803.xml]: TODO to me to create this, or just remove the warning

Comment (Ericson2314) [reply nixos/doc/manual/release-notes/rl-1803.xml]: Motivation: For another different one, I want to hyperlink a nixpkgs manual section, which is only readily done from within the nixpkgs manual.

Comment (globin) [reply nixos/doc/manual/release-notes/rl-1803.xml]: I think we've intentionally restricted nixpkgs/NixOS release notes to NixOS as there is hardly any code which could not be added to both and we don't technically have releases for nixpkgs,

**Review by globin — APPROVED** [review]

File: nixos/doc/manual/release-notes/rl-1803.xml (hunk 1 above)
Comment (vcunat) [reply nixos/doc/manual/release-notes/rl-1803.xml]: The same argument would seem to work for merging the two manuals :-)

Comment (globin) [reply nixos/doc/manual/release-notes/rl-1803.xml]: I think I'd agree, simply by looking at my behaviour of grepping through both every time I look for something. But that discussion should not start here, but rather on the mailing list or an RFC. :)

**Pull request merged**